docs/modules/operation/pages/events/sources/snmp-traps.adoc

```diff
@@ -141,7 +142,7 @@ You can use the `parm` replacement token to access them.
 Each parameter consists of a name and a value.
 
 For example, the `ciscoC3800SysAggregateStatusChange` event description lists out each of the parameters.
-Thus the second paramater, the `sysName`, is printed using `%parm[#2]%`.
+Thus the second parameter, the `sysName`, is printed using `%parm[#2]%`.
```

Comment:
Line 147 ... link should goes to wrong location. Should be "parameter tokens" in the event-tokens.adoc file.